plugins/inputs/aerospike/aerospike.go

```diff
@@ -24,7 +27,7 @@ var sampleConfig = `
   ## Aerospike servers to connect to (with port)
   ## This plugin will query all namespaces the aerospike
   ## server has configured and get stats for them.
-  servers = ["localhost:3000"]
+  servers = ["user:password@localhost:3000","localhost:3000"]
```

Comment:
don't add another server to the list, just document in a comment that this is how to format user/pass combos